net/http/http_cache_transaction.cc

```diff
@@ -4071,6 +4071,21 @@ bool HttpCache::Transaction::ShouldDisableCaching(
     return true;
   }
 
+#if BUILDFLAG(IS_STARBOARD)
```

Comment:
This would not be included for Android builds -  please see [go/cobalt-naming](https://goto.google.com/cobalt-naming)

To enable this in all Cobalt builds please use `BUILDFLAG(IS_COBALT)`